src/Dropdown/Dropdown.mdx

```diff
@@ -114,6 +115,7 @@ Easily display contextual overlays using custom trigger elements. Dropdown's pos
               </Dropdown>
             </Flex>
           </Button.Group>
+          <Input name="password" label="Password" />
```

Comment:
This seems like an existing issue and may be out of scope but I have to click twice into an input field when the dropdown is open in order to get the input field to focus.

Reply:
@erikshestopal this PR is actually addressing that issue
@cehsu experiencing the same as erik - clicking into the input does not focus the input

Reply:
That's correct, the original bug was that the dropdown would not close when an input was clicked. Now the dropdown will close when an input is clicked, and the input can be subsequently focused.

Reply:
Updated to reimplement FocusLock's `returnFocus`, which was preventing the input focus on blur.